### DISCUSSION TITLE
Best way to save Atom files?

### DISCUSSION THREAD

**Discussion by nhudgins** [community/community#35007] [New to GitHub]
Hello - brand new to coding (student).  I tried to save a style.css file from Atom and all that that was "saved" was a blank Notepad file.  Can you offer thoughts on what I did wrong?  Is it possible to save an Atom file directly to GitHub online?  I couldn't see how to do that.  Thanks for your kindness!

**Comment by DSDmark** [comment]
Hey @nhudgins,

Try to write what specific problem you face.
About your **style** file, did you save by Ctrl + S or Ctrl+Shift + S 
if it is not work, then try to change it in **Preferences** setting.

##  About GitHub 
if you want to upload file or folder on GitHub then add **extension** it will help you to push your repo on git
and you can read that article 
> [Click On It](https://courses.cs.washington.edu/courses/cse154/19su/resources/assets/atomgit/windows/ "git artical")
